docs/ISuite/50-Development/add-information-to-the-message-processing-log-e8e9283.md

```diff
@@ -52,12 +52,8 @@ You can apply the following method on an instance of `MessageLog` in order to cr
 The following Groovy Script stores the message body in an MPL attachment \(`My Attachment`\):
 
```

Comment:
```suggestion
def Message processData(Message message) {
    def body = message.getBody(java.lang.String)
    def messageLog = messageLogFactory.getMessageLog(message)
    if (messageLog != null) {
        messageLog.addAttachmentAsString('My Attachment', body, 'text/plain')
    }
    return message
}
You can simplify this expression in the following way using idiomatic Groovy code and style:
Message processData(Message message) {
    messageLogFactory.getMessageLog(message)?.addAttachmentAsString('My Attachment', message.getBody(String), 'text/plain')
    return message
}
```

Reply:
Thanks for this great feedback! I have discussed this with some experts, and we propose to keep the original code sample and add your simplified version as an additional option.